cosmovisor/upgrade.go

```diff
@@ -122,7 +122,10 @@ func GetDownloadURL(info *UpgradeInfo) (string, error) {
 	if err == nil {
 		url, ok := config.Binaries[osArch()]
 		if !ok {
-			return "", errors.Errorf("cannot find binary for os/arch: %s", osArch())
+			url, ok = config.Binaries["*"]
```

Comment:
I'd use a keyword instead of the wildcard `*` character as it simplifies parsing and avoids potential complications with regexes:

```suggestion
			url, ok = config.Binaries["any"]
```